docs/functionalities/core-functionalities.md

```diff
@@ -3,23 +3,25 @@ id: core-functionalities
 title: Core Functionalities
 ---
 
-
->The current Talawa app is functional but does not meet many best practices. We need to refactor the code to be internationally acceptable. 
+> The current Talawa app is functional but does not meet many best practices. We need to refactor the code to be internationally acceptable.
 
 > Before we can add any new functionality. The refactored app needs to meet these basic requirements on which the original application was based.
 
 ## User Registration
+
 ### Description
-This is the ability to register in the app with an organisation. Once the user is successfully registered as a member of an organisation they are then able to interact with, app events and other content related to the organisation.
+
+This is the ability to register in the app with an organization. Once the user is successfully registered as a member of an organization they are then able to interact with, app events and other content related to the organization.
```

Comment:
Please change to: 
This is the ability to register in the app with an organization. Once the user is successfully registered as a member of an organization, they are then able to interact with app events and other content related to the organization.